internal/buildscripts/packaging/choco/splunk-otel-collector/splunk-otel-collector.nuspec

```diff
@@ -0,0 +1,50 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>splunk-otel-collector</id>
+    <version>0.0.0</version>
+    <description>
+The splunk-otel-collector is a metric-based agent written in Go that is used to monitor infrastructure and application services from a variety of environments.
+
+The splunk-otel-collector looks for a configuration file at `\ProgramData\Splunk\OpenTelemetry Collector\otelcol.exe`.  If this file does not already exist during installation, a default config file will be copied into place by the installer.
+
+#### Package Parameters
+The following package parameters are available:
+
+ * `/SPLUNK_ACCESS_TOKEN` - The splunk access token (org token) used to send metric data to OTEL. If the parameter is specified, the token will be saved to the `\ProgramData\Splunk\OpenTelemetry Collector\token` file. If the parameter is not specified and `\ProgramData\Splunk\OpenTelemetry Collector\token` does not exist or is empty, the Agent service is not started after installation or upgrade. To start the service, add or update `\ProgramData\Splunk\OpenTelemetry Collector\token` with a valid token, and then either restart Windows or run the following PowerShell command: `&amp; "\ProgramData\Splunk\OpenTelemetry Collector\otelcol.exe"`
+ * `/SPLUNK_INGEST_URL:` - URL of the Splunk ingest  (e.g. `https://ingest.YOUR_SPLUNK_REALM.signalfx.com`). The URL will be saved to the `\ProgramData\Splunk\OpenTelemetry Collector\SPLUNK_INGEST_URL` file. If the parameter is not specified, the value found in `\ProgramData\Splunk\OpenTelemetry Collector\SPLUNK_INGEST_URL` (if it exists) will be used. Otherwise, defaults to `https://ingest.us0.signalfx.com`.
+ * `/PLUNK_API_URL` - URL of the API endpoint (e.g. `https://api.YOUR_SPLUNK_REALM.signalfx.com`). The URL will be saved to the `\ProgramData\Splunk\OpenTelemetry Collector\SPLUNK_API_URL` file. If the parameter is not specified, the value found in `\ProgramData\Splunk\OpenTelemetry Collector\SPLUNK_API_URL` (if it exists) will be used. Otherwise, defaults to `https://api.us0.signalfx.com`.

```

Comment:
```suggestion
 * `/SPLUNK_API_URL` - URL of the API endpoint (e.g. `https://api.YOUR_SPLUNK_REALM.signalfx.com`). The URL will be saved to the `\ProgramData\Splunk\OpenTelemetry Collector\SPLUNK_API_URL` file. If the parameter is not specified, the value found in `\ProgramData\Splunk\OpenTelemetry Collector\SPLUNK_API_URL` (if it exists) will be used. Otherwise, defaults to `https://api.us0.signalfx.com`.

```